### DISCUSSION TITLE
Suppressing output from FFmpegFrameGrabber

### DISCUSSION THREAD

**Discussion by Haprst** [bytedeco/javacv#2204] [Q&A] [answered]
I'm writing a JavaCV demo than analyzes a 5 minute video with about 10,000 frames. For each frame, FFmpegFrameGrabber outputs about 10 lines of metadata that is of absolutely no interest to me. That adds up to 100,000 lines of output that makes it somewhat difficult to spot my own debugging output that I need to find. The question is how to suppress generating all that unneeded output. I found two places on the web that purport to answer this question, but both answers failed to work, presumably because they are over 5 years old. I'm hoping that someone can provide a current answer that actually works.

**Comment by saudet** [comment]
Please call something like `av_log_set_level(AV_LOG_ERROR);`

**Comment by saudet** [comment]
Duplicate of https://github.com/bytedeco/javacv/issues/1147

**Comment by Haprst** [comment, marked as answer]
Before posting this problem, I had already tried avutil.av_log_set_level(avutil.AV_LOG_QUIET); but the suggested import location was wrong. Based on your advice, I was able to discover that the correct location is import org.bytedeco.ffmpeg.global.avutil; (I'm somewhat surprised that NetBeans didn't know that.) So, indirectly you solved my problem. Thank you. But the correct parameter is AV_LOG_QUIET not AV_LOG_ERROR.
